doc/classes/File.xml

```diff
@@ -83,7 +83,7 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the whole file as a [String].
+				Returns the whole file as a [String]. Text is interpreted as being UTF-8 encoded.
```

Comment:
This should be put below the existent text, to keep it consistent with the rest of the changes.